CONTRIBUTING.md

```diff
@@ -73,7 +73,7 @@ Please see our [Style Guide](STYLE_GUIDE.md) for more details.
 Nighly CI tests are ran and results can be found on the central README. To 
 subscribe for alerts please join the [addons-testing mailing list](https://groups.google.com/a/tensorflow.org/forum/#!forum/addons-testing).
 
-#### Locally Testing
+#### Locally Testing CPU
 
 ```bash
 docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:nightly-custom-op make unit-test
```

Comment:
`bash tools/run_docker.sh -c 'make unit-test'`

Reply:
And we also can revise Development Environment section above:
```
Try these commands below:
* Format code automatically:  bash tools/run_docker.sh -c 'make code-format'
... ...
```

Reply:
Good suggestions... thanks! Updated.